app/controllers/webhooks_controller.rb

```diff
@@ -66,6 +66,12 @@ def mandrill
     success
   end
 
+  def mandrill_head
+    # Mailchimp sends a HEAD request to validate the webhook before saving
```

Comment:
the fix in theory looks fine, but having Mailchimp in a Mandrill route is very confusing

Reply:
Yeah looking at it again it made little sense, can change it to mailchimp again if/when they finally migrate transactional.